src/guides/wallet.md

```diff
@@ -0,0 +1,164 @@
+By using the HD standards mentioned above, hsd allows one to easily restore or transfer their entire wallet to different wallet implementations. By providing just the mnemonic, one can fully recover their wallet to a fresh instance of `hsd` or any other software that properly implements BIP33, BIP39, and BIP44, like the Trezor hardware wallet.
+
+*Above borrowed from [Wallets and Accounts and Keys, Oh My!](https://bcoin.io/guides/wallets.html) by Daniel McNally*
+&nbsp;\
+&nbsp;\
+
+<h2 id='examples'>Examples</h2>
+
+Command line examples against a local `hsd` server.
+
+To begin, install [hsd]([https://handshake-org.github.io/) and [hs-client](https://github.com/handshake-org/hs-client)
+&nbsp;\
+&nbsp;\
+
+### hs-client: Interact with your wallet
+
+`hs-client` comes with four executables. For the Wallet you will want to use `hsw-cli` and `hsw-rpc`.
+&nbsp;\
+
+* `hsw-cli` - Wallet REST API
+&nbsp;\
+
+* `hsw-rpc` - Wallet RPC interface 
+&nbsp;\
+
+* `hsd-rpc`  - hsd node RPC interface
+&nbsp;\
+
+* `hsd-cli` - hsd node REST API
+&nbsp;\
+
+
+For more info on node commands unrelated to the wallet. [See here](https://handshake-org.github.io/api-docs))
+
+Below you will find short intros to both wallet tools `hsw-cli` and `hsw-rpc` Use them for wallet creation, sending coins, generating addresses, bidding on names, updating resource records for a name, transfering names, etc.. 
+&nbsp;\
+
+
+### Set the network option
+
+You have four choices to chose when setting your `hsd` node's network. The `hs-client` CLI tools also require this option be set for them to know the port with which to connect to your `hsd` node.
+
+* `regtest` Best for local development work.
+* `testnet` Closest simulation to mainnet. Supported by a network of nodes run by other devs.
+* `mainnet` Live production.
+
+You can set the network for your hsd node with the `--network=` option. Or with `$ export HSD_NETWORK=` 
+Choose 'regtest' for testing out the commands below.
+
+
+### hsw-cli
+
+&nbsp;\
+Create a new wallet and encrypt it with a passphrase 
+``` bash
+$ hsw-cli mkwallet mynewwallet --passphrase=supersecret
+```
+
+Response JSON:
+``` json
+{
+  "network": "regtest",
+  "wid": 3,
+  "id": "ha",
+  "watchOnly": false,
+  "accountDepth": 1,
+  "token": "aef7603ff78c32e267fad434246ce6447d420b81e4798d99cf06c80b57c40765",
+  "tokenDepth": 0,
+  "master": {
+    "encrypted": false
+  },
+}
+```
+
+&nbsp;\
+See that `token`? With it we can do things like query our wallet balance
+``` bash
+$ hsw-cli balance --id=mynewwallet --token=aef7603ff78c32e267fad434246ce6447d420b81e4798d99cf06c80b57c40765
+```
+
+&nbsp;\
+Generate a new receiving address for an account
+``` bash
+$ hsw-cli --id=$name-of-wallet --accout=$name-of-account address
+```
+
+&nbsp;\
+Get the HNS balance of your wallet
+``` bash
+$ hsw-cli --id=balance
+```
+
+&nbsp;\
+List all wallets in your walletdb.
+``` bash
+$ hsw-cli wallets
+```
+
+&nbsp;\
+Send HNS coins to an address
+``` bash
+$ hsw-cli send 
```

Comment:
This is not correct. There are other arguments that need to be specified to send.